troubleshooting/intel-igfx-troubleshooting.md

```diff
@@ -5,9 +5,25 @@ permalink: /doc/intel-igfx-troubleshooting/
 ---
 # Intel Integrated Graphics Troubleshooting #
 
-Dom0 Kernels currently included in Qubes have issues related to VT-d (IOMMU) and some versions of the
-integrated Intel Graphics Chip. Depending on the specific hardware / software combination the issues are quite wide ranging, from apparently harmless log errors, to VM window refresh issues,
-to complete screen corruption and crashes rendering the machine unusable with Qubes.
+## Software Rendering or Video Lags
+
+If you are experiencing this issue, you will see extremely slow graphics updates.
+You will be able to watch the screen and elements paint slowly from top to bottom.
+You can confirm this is the issue by looking for a line similar to the following in your `/var/log/Xorg.0.log` file:
+
+    [   131.769] (EE) AIGLX: reverting to software rendering
+
+Newer versions of the Linux kernel have renamed the `i915.preliminary_hw_support=1` option to `i915.alpha_support=1`, so if you needed this kernel option in the past you will have to rename it or add it to your configuration files (follow either GRUB2 or EFI, not both):
```

Comment:
Oh, thanks. I think we may want to enable it by default - same as for `i915.preliminary_hw_support=1`.